build/build.xml

```diff
@@ -277,6 +277,43 @@
 			</else>
 		</if>
 
+		<!-- Checkout latest viglink to ext folder -->
+		<available file="${dir}/ext" type="dir" property="add-viglink-ext" />
+		<exec dir="${dir}"
+			command='php -r "echo version_compare(&apos;${version}&apos;, &apos;3.2.0-b3&apos;, &apos;>=&apos;) ? &apos;true&apos; : &apos;false&apos;;"'
+			checkreturn="true"
+			outputProperty='viglink-available' />
+		<if>
+			<and>
+				<equals arg1="${add-viglink-ext}" arg2="1" trim="true" />
+				<or>
+					<equals arg1="${revision}" arg2="HEAD" trim="true" />
+					<equals arg1="${viglink-available}" arg2="1" trim="true" />
+				</or>
+			</and>
```

Comment:
If I understand correctly, viglink will be present in the package if the version is >= 3.2.0-b3 or if viglink is present in the directory?

I'm not sure that's what we want.

IIRC, we said: always on install
For the update I don't remember, and I don't really know what to do:
 - I wouldn't reinstall it (even if not enabled) if the user have deleted the files
 - I wouldn't update the files and create conflicts if the user has updated (or reinstalled) the extension manually
 - Could be nice to update the extension if the user didn't touch it...

So... I don't know